### PR TITLE
fix search link icon to respect the selected index field

### PR DESCRIPTION
**Pull request by petschki**

### CODE REVIEW

File: Products/PloneKeywordManager/browser/prefs_keywords_view.pt [hunk 1]
```diff
@@ -177,11 +177,11 @@
                                  onclick string:document.forms['keyword_edit_form'].changeto.value='${subject}';; return true;;" />
           <label tal:content="subject" tal:attributes="for string:subject-${subject_id};">Subject</label>
           <tal:p5 tal:condition="view/is_plone_5">
-            <a href="#" tal:attributes="href string:${portal_url}/@@search?Subject=${subject}">
+            <a href="#" tal:attributes="href string:${portal_url}/@@search?${field}=${subject}" target="_blank">
```
Comment (hvelarde) [inline Products/PloneKeywordManager/browser/prefs_keywords_view.pt]: why `target="_blank"`?

**Comment by hvelarde** [comment]
CI seems to be broken; someone needs to fix this before merging.

**Comment by petschki** [comment]
@hvelarde there is a version conflict in https://github.com/collective/buildout.plonetest/blob/master/qa.cfg
the `coverage` egg is pinned in ztk versions
```
coverage= 3.5.2
    http://dist.plone.org/versions/zopetoolkit-1-0-8-ztk-versions.cfg
```
but the `createcoverage` egg has a minimum dependency of `coverage >= 3.7`
@reinout where do we have to solve this? https://github.com/collective/buildout.plonetest ?

File: Products/PloneKeywordManager/browser/prefs_keywords_view.pt (hunk 1 above)
Comment (petschki) [inline Products/PloneKeywordManager/browser/prefs_keywords_view.pt]: usually you want to look quickly, which articles are affected by this subject. so we found it useful to open this quick review in a new window.

**Comment by hvelarde** [comment]
@petschki I fixed the buildout; please rebase your branch.

File: Products/PloneKeywordManager/browser/prefs_keywords_view.pt (hunk 1 above)
Comment (hvelarde) [inline Products/PloneKeywordManager/browser/prefs_keywords_view.pt]: in general, opening a new window is a basic usability mistake that must be avoided: see item 9 of: https://www.nngroup.com/articles/top-10-mistakes-web-design/

opening a modal would be better.

**Comment by petschki** [comment]
@hvelarde all green now

**Pull request merged**